src/quo2/components/list_items/received_contact_request.cljs

```diff
@@ -0,0 +1,79 @@
+(ns quo2.components.list-items.received-contact-request
+  (:require [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [status-im.utils.handlers :refer [<sub >evt]]
+            [quo2.components.avatars.user-avatar :as user-avatar]
+            [quo2.foundations.typography :as typography]
+            [clojure.string :as str]
+            [status-im.utils.utils :as utils.utils]
+            [status-im.utils.datetime :as time]
+            [status-im.i18n.i18n :as i18n]))
+
+(defn get-display-name [chat-id no-ens-name no-nickname]
+  (let [name (first (<sub [:contacts/contact-two-names-by-identity chat-id]))]
+    (if (and no-ens-name no-nickname)
+      (str (first (str/split name " ")) " " (second (str/split name " ")))
+      name)))
+
+(defn list-item [{:keys [chat-id image contact message timestamp]}]
+  (let [no-ens-name  (str/blank? (:ens-name (:content message)))
+        no-nickname  (if (= (:nickname (:names contact)) nil) true false)
```

Comment:
You could refactor this conditional to be a bit clearer by using `get-in` and the core `nil?` function:

```clojure
(nil? (get-in contact [:names :nickname]))
```